docs/assets/css/styles.scss

```diff
@@ -120,9 +138,9 @@ pre>code {
     }
 }
 
-code {
-    color: #545454;
-}
+// code {
+//     color: #545454;
+// }
```

Comment:
I commented this out so that the in-line code-blocks were also colored (the default pink I believe), but please undo/remove as appropriate!

Reply:
Yes, this was intentional to overwrite the default red as it's low contrast. It does not meet accessibility.

Reply:
Aha gotcha, I will undo!

Reply:
I'll have to go in and customize some code colors in `.highlight` to make them accessible. 

![Screen Shot 2021-09-15 at 1 26 02 PM](https://user-images.githubusercontent.com/22301405/133504493-717dccde-059c-4a72-b09d-8eca24a6d247.png)

Reply:
Totally, all good 👍🏽